receiver/simpleprometheusreceiver/receiver.go

```diff
@@ -60,8 +61,30 @@ func (prw *prometheusReceiverWrapper) Start(ctx context.Context, host component.
 	return prw.prometheusRecever.Start(ctx, host)
 }
 
+// Deprecated: [v0.55.0] Use getPrometheusConfig instead.
+func getPrometheusConfigWrapper(cfg *Config, params component.ReceiverCreateSettings) (*prometheusreceiver.Config, error) {
+	if cfg.TLSEnabled {
+		params.Logger.Warn("the `tls_config` and 'tls_enabled' settings are deprecated, please use `tls` instead")
+		cfg.HTTPClientSettings.TLSSetting = configtls.TLSClientSetting{
+			TLSSetting: configtls.TLSSetting{
+				CAFile:   cfg.TLSConfig.CAFile,
+				CertFile: cfg.TLSConfig.CertFile,
+				KeyFile:  cfg.TLSConfig.KeyFile,
+			},
+			InsecureSkipVerify: cfg.TLSConfig.InsecureSkipVerify,
+		}
+	}
+	return getPrometheusConfig(cfg)
+}
+
 func getPrometheusConfig(cfg *Config) (*prometheusreceiver.Config, error) {
-	var bearerToken string
+	var (
+		bearerToken string
+		scheme      = "http"
+		out         = &prometheusreceiver.Config{}
+		httpConfig  = configutil.HTTPClientConfig{}
+	)
```

Comment:
I would avoid using this style inside functions. I doesn't improve readability and moves variable definitions farther away from its usage. Please revert this block